2022-12-16 10:38:42,706 — ERROR : Unknown error
Traceback (most recent call last):
  File "/Users/junfeng/Documents/WorkSpace/Python/IntroProgramming/emergency-system/get.py", line 48, in listing
    arr[i] = int(arr[i])
ValueError: invalid literal for int() with base 10: '1 2 3 4 5 6 7'
2022-12-16 10:38:46,290 — ERROR : Unknown error
Traceback (most recent call last):
  File "/Users/junfeng/Documents/WorkSpace/Python/IntroProgramming/emergency-system/get.py", line 48, in listing
    arr[i] = int(arr[i])
ValueError: invalid literal for int() with base 10: '1 2 3'
2022-12-16 10:39:33,239 — ERROR : You entered a non-numeric value.
Traceback (most recent call last):
  File "/Users/junfeng/Documents/WorkSpace/Python/IntroProgramming/emergency-system/get.py", line 16, in int
    n = int(input(hint))
ValueError: invalid literal for int() with base 10: '???'
2022-12-16 10:39:33,240 — ERROR : Please reenter a valid value.
Traceback (most recent call last):
  File "/Users/junfeng/Documents/WorkSpace/Python/IntroProgramming/emergency-system/get.py", line 16, in int
    n = int(input(hint))
ValueError: invalid literal for int() with base 10: '???'
2022-12-16 11:28:54,876 — ERROR : You entered a non-numeric value.
Traceback (most recent call last):
  File "/Users/junfeng/Documents/WorkSpace/Python/IntroProgramming/emergency-system/myfunctionlib.py", line 102, in menu_choice_get
    option = int(input(hint))
ValueError: invalid literal for int() with base 10: 'q'
2022-12-16 11:28:54,879 — ERROR : Please reenter a valid Number
Traceback (most recent call last):
  File "/Users/junfeng/Documents/WorkSpace/Python/IntroProgramming/emergency-system/myfunctionlib.py", line 102, in menu_choice_get
    option = int(input(hint))
ValueError: invalid literal for int() with base 10: 'q'
2022-12-16 11:28:55,220 — ERROR : You entered a non-numeric value.
Traceback (most recent call last):
  File "/Users/junfeng/Documents/WorkSpace/Python/IntroProgramming/emergency-system/myfunctionlib.py", line 102, in menu_choice_get
    option = int(input(hint))
ValueError: invalid literal for int() with base 10: 'q'
2022-12-16 11:28:55,221 — ERROR : Please reenter a valid Number
Traceback (most recent call last):
  File "/Users/junfeng/Documents/WorkSpace/Python/IntroProgramming/emergency-system/myfunctionlib.py", line 102, in menu_choice_get
    option = int(input(hint))
ValueError: invalid literal for int() with base 10: 'q'
